### DISCUSSION TITLE
[DASHBOARD] Adjust the position of key display for items, fill character, and a question about recent files

### DISCUSSION THREAD

**Discussion by farzadmf** [folke/snacks.nvim#204] [Q&A]
I'm going all-in with the dashboard, and I'm loving it.

Using mini.starter before, I had something like this:
![image](https://github.com/user-attachments/assets/ab56761a-5069-4941-b1dd-62979c819df8)

And with Snacks, I have this (items are a bit different as it's WIP):
![image](https://github.com/user-attachments/assets/519f53f1-0b85-4265-ab11-d06bbc5bbb2e)

I use the recent files section quite a lot, and I had a few questions:
1. In mini.starter, I _think_ (not sure TBH), listed recent files are only applicable to the current project/directory; is there a way to achieve the same with snacks?
2. In mini.starter, the key is displayed to the left of the item, so when I locate the file, I can quickly recognize its key, but in Snacks (and other dashboards), I have to gaze through to the right to figure out the key.

Next question is mainly applicable if second question above doesn't work:
3. Is there a way to somehow display some kind of a "fill character" in the space between the file and its key? Something like:
```
Recent files
  options.txt (/o/h/C/n/0/s/n/r/d) .............................. [4]
  other.txt (/my/dir) ........................................... [4]
...
```

**Comment by folke** [comment]
1. add `pwd = true` to the recent files section
2. keys are always on the right
3. you can just add `...` to the item desc

**Reply by farzadmf** [reply]
1. Thank you!
2. Could this be something you're open to add at some point? (configurable in `opts` for example)
3. The issue with me adding them manually would be that they wouldn't properly fill the whole empty space, would they?

EDIT:
1. I think it's `cwd = true` and not `pwd = true`
